✨(richie) add richie_init job

jmaupetit opened the pull request:
## Purpose

While bootstrapping a new Richie instance, one need to create a base pages tree thanks to the `richie_init` management command.

## Proposal

- [x] add `richie-init` job 
- [x] execute the `richie-init` job before bootstrapping elasticsearch

jmaupetit committed:
✨(richie) add richie_init job

While bootstraping a new Richie instance, one need to create a base
pages tree thanks to the richie_init management command.

It appears to make sense to execute this job before bootstraping
elasticsearch.

sampaccoud commented on apps/richie/templates/services/app/job_03_richie_init.yml.j2:
```diff
@@ -0,0 +1,49 @@
+              value: "richie-{{ richie_database_host }}-{{ deployment_stamp }}"
+            - name: DB_PORT
+              value: "{{ richie_database_port }}"
+            - name: DJANGO_ALLOWED_HOSTS
```
not required for a job?

jmaupetit replied:
True.

jmaupetit committed:
🔥(richie) remove DJANGO_ALLOWED_HOSTS jobs env var

It is not required to set the DJANGO_ALLOWED_HOSTS environment
variable/setting to run a Django management command in a job.

jmaupetit committed:
👷(richie) fix test url

When a Richie project has been bootstrapped, it does not have a root
page yet. Thus, it is more consistent to test the /admin url that must
always respond whatever the instance status is.
